src/server/bg_services/replication_scanner.js

```diff
@@ -194,6 +232,43 @@ class ReplicationScanner {
         }
     }
 
+    // list_objects_versions will list all the objects with the versions, continuation_token is the key marker.
+    async list_objects_versions(bucket_name, prefix, continuation_token) {
```

Comment:
add here a comment that we prefer keeping the cont_token term due to re-use of db properties but this is really a key marker

Reply:
Sure, will add a comment on the next PR.